src/MultiSelect/MultiSelectItem.js

```diff
@@ -0,0 +1,24 @@
+import React from "react";
+import PropTypes from "prop-types";
+
+const MultiSelectItem = ({ children }) => <>{children}</>;
```

Comment:
This component gathers information about items so we can have an interface like this:

```jsx
<MultiSelect>
    <MultiSelect.Item>My item jsx</MultiSelect.Item>
</MultiSelect>
```

Instead of accepting a big data structure like this: `<MultiSelect options={{ ...bigJsObject }} />`.

I'm exposing this `Thing.Item` interface for two reasons:
- This would be consistent with other components, like `Select` and `Tabs`
- This gives consumers full control over what gets rendered in each item